modules/appeals_api/config/schemas/v2/200996.json

```diff
@@ -79,7 +79,8 @@
     "hlrCreateRepresentative": {
```

Comment:
Should this also specify that we're talking about the informal conference rep?

Reply:
Yeah. Let's do it.